doc/releases/changelog-0.42.0.md

```diff
@@ -246,8 +288,11 @@
 
 <h4>OpenQASM 🤝 PennyLane</h4>
 
-* A new function called :func:`qml.from_qasm3` has been added, which converts OpenQASM 3.0 circuits into quantum functions
-  that can be subsequently loaded into QNodes and executed. 
+PennyLane now offers improved support for [OpenQASM](https://openqasm.com/),
+including the ability to convert between PennyLane and OpenQASM circuits.
```

Comment:
Slight nuance here! PL has had QASM 2.0 functionality for a while (see https://docs.pennylane.ai/en/stable/code/api/pennylane.from_qasm.html). The functionality in this release is partially for 3.0 and 2.0 support. To avoid confusion, I might just remove the opening statement and let the individual entries to the talkin'

```suggestion
```